app/javascript/components/applications/application_form/PaperApplicationForm.js

```diff
@@ -53,6 +54,22 @@ class PaperApplicationForm extends React.Component {
       errors.alternate_contact.last_name = validate.isPresent('Please enter a Last Name')(values.alternate_contact.last_name)
       errors.alternate_contact.email = validate.isValidEmail('Please enter a valid Email')(values.alternate_contact.email)
     }
+
+    // Secondary preference application member validation: Check if selected app member value is still a valid natural key.
+    if (values.preferences) {
+      // Combine household members with primary applicant to get list of all the natural keys
+      const allMembers = [values.applicant].concat(values.household_members ? values.household_members : [])
+      const naturalKeys = allMembers.map((member) => naturalKeyFromMember(member))
+      errors.preferences = []
+      values.preferences.map(
+        (pref, i) => {
+          errors.preferences = errors.preferences.concat({})
+          if (pref.naturalKey && !includes(naturalKeys, pref.naturalKey)) {
+            errors.preferences[i].naturalKey = 'This field is required'
```

Comment:
Note, the required message actually varies for all of these fields generally, but I wasn't able to get this validation to work sufficiently at the field level so had to bump up to form-level. 

I think we could: Keep message as "This field is required" in this use case, or set the error messages at a higher level and try to match the validation message to the given preference.